### PR TITLE
add crypto-bot

### PR DESCRIPTION
**Pull request by Voyager-One**

### CODE REVIEW

File: apps/crypto-bot/crypto-bot.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Crypto-Bot
+description: 'app for encryption and decryption'
```
Comment (zeke) [inline apps/crypto-bot/crypto-bot.yml]: No need to put "app" in the description. Also is it for encrypting files? Messages across a network?

Comment (Voyager-One) [reply apps/crypto-bot/crypto-bot.yml]: no network request. It works locally

Comment (zeke) [reply apps/crypto-bot/crypto-bot.yml]: Got it. Please update the description so this is clear to everyone.

**Comment by Voyager-One** [comment]
####it's like this : )
image
image

**Pull request merged**